framework/include/base/XFEMInterface.h

```diff
@@ -54,6 +55,8 @@ class XFEMInterface : public ConsoleStreamInterface
     _fe_problem(params.getCheckedPointerParam<FEProblemBase *>("_fe_problem_base")),
     _material_data(NULL),
     _bnd_material_data(NULL),
+    _moose_mesh(NULL),
+    _moose_mesh2(NULL),
```

Comment:
`_moose_mesh2`? This isn't a very descriptive variable name. Why are there two? Can you name it something better?

Unrelated, we prefer `nullptr` over `NULL` in C++11 because of places where we infer type. NULL is actually an `int`, feel free to replace `NULL` when you see it, and avoid adding new ones.